doc/rest_api/snippet_sharing/use_cases/UC_delete_snippet.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: delete configuration snippet
+Scope: Entry Management
+Level: User Goal
+Actors: Authenticated user, Moderator
+Brief: Users can delete their own database entries the same way that moderators can delete any configuration snippet in the database.
+
+Scenarios:
+----------------------------
+Precondition: Authenticated, sufficient permissions
+Main success scenario: Configuration snippet has been deleted successfully. A success response is returned.
+Alternative scenario: The specified snippet cannot be found (wrong URI) or the user has insufficient permissions to delete it. Either way the user is informed about the error by a response.
+Error scenario: Technical problems prevent the deletion process from completing. The user is informed by an error response.
+Postcondition: Configuration snippet has been deleted from the database and can neither be found through search, nor be viewed in detail anymore.

```

Comment:
What about some "undo" feature? (Can be implemented as backup of the whole key database, e.g. in git)